app/src/main/java/com/example/tasky/actions/presentation/ActionRoot.kt

```diff
@@ -26,40 +27,45 @@ import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
 import androidx.navigation.NavController
 import com.example.tasky.R
+import com.example.tasky.actions.presentation.components.EmptyPhotos
 import com.example.tasky.common.domain.util.convertMillisToDate
-import com.example.tasky.common.presentation.CheckboxHeader
 import com.example.tasky.common.presentation.DateLineItem
 import com.example.tasky.common.presentation.GrayDivider
 import com.example.tasky.common.presentation.RightCarrotIcon
-import com.example.tasky.event.presentation.components.EmptyPhotos
+import com.example.tasky.common.presentation.TitleSection
+import com.example.tasky.common.presentation.model.Action
 
 @Composable
-fun EventRoot(
+fun ActionRoot(
```

Comment:
What does ActionRoot mean? Why call this action? 🤔 

We're talking about agenda details here after all, right?

Reply:
yeah! I can rename